app/src/components/InputElement/InputElement.styles.js

```diff
@@ -3,12 +3,35 @@ import background from "../../images/bg-shorten-desktop.svg";
 
 export const StyledForm = styled.form`
   position: absolute;
-  left: 270px;
+  left: 128px;
+  top:705px;
   border: 1px solid ${(props)=>props.theme.colors.black};
+  border-radius: 5px;
   display: flex;
   align-items: center;
   justify-content: center;
-  height: 110px;
-  width: 900px;
+  height: 130px;
```

Comment:
Please use rem units to set height

Reply:
corrected. sorry I forgot to ask also about height not only width.